pytravharv/__main__.py

```diff
@@ -45,119 +43,49 @@ def get_arg_parser():
         "-n",
         "--name",
         type=str,
+        required=False,
         default=None,
         help="Name of the configuration to use",
     )
 
+    parser.add_argument(
+        "-m",
```

Comment:
does not feel like `--mode`?

why not `--store` taking two required arguments read_uri, write_uri

and have it default in a smart way to 
- uristore if .env has READ_URI and WRITE_URI set
- memorystore if not